src/dhcpcd.c

```
@@ -1575,34 +1575,40 @@ dhcpcd_signal_cb(int sig, void *arg)
 	/* Make any change here in privsep-control.c as well. */
 	if (strcmp(*argv, "--version") == 0) {
-		return control_queue(fd, UNCONST(VERSION), strlen(VERSION) + 1);
+		return control_queue(fd, VERSION, strlen(VERSION) + 1);
 	} else if (strcmp(*argv, "--getconfigfile") == 0) {
-		return control_queue(fd, UNCONST(fd->ctx->cffile),
+		return control_queue(fd, fd->ctx->cffile,
 		    strlen(fd->ctx->cffile) + 1);
 	} else if (strcmp(*argv, "--getinterfaces") == 0) {
 		oifind = argc = 0;
-		goto dumplease;
+		opts |= DHCPCD_DUMPLEASE;
```

Cppcheck (2.21.0)

[error] Uninitialized variable

(legacyUninitvar)